renku/core/models/workflow/plan.py

```diff
@@ -64,20 +147,13 @@ def __init__(
         outputs: List[CommandOutput] = None,
         success_codes: List[int] = None,
     ):
+        super().__init__(id=id, description=description, invalidated_at=invalidated_at, keywords=keywords, name=name)
```

Comment:
since python allows `super` to be called anywhere within the ctor, this should be moved on the end of the ctor as currently if the Plan init is called without setting `name` param, the init in `AbstractPlan` will call `AbstractPlan::_get_default_name` that'll call `Plan::to_argv`, which params are not yet set, because `AbstractPlan::__init__` runs before setting these params, like `Plan.inputs`, `Plan.outputs` etc.